model_api/python/openvino/model_api/models/utils.py

```diff
@@ -88,17 +82,26 @@ def __init__(self, segmented_object):
         )
         self.rotated_rects = []
 
-    def __to_str(self):
-        res = f"({self.xmin}, {self.ymin}, {self.xmax}, {self.ymax}, {self.score:.3f}, {self.id}, {self.str_label}, {(self.mask > 0.5).sum()}"
+    def __str__(self):
+        res = super().__str__()
         for rect in self.rotated_rects:
             res += f", RotatedRect: {rect[0][0]:.3f} {rect[0][1]:.3f} {rect[1][0]:.3f} {rect[1][1]:.3f} {rect[2]:.3f}"
-        return res + ")"
+        return res
 
-    def __str__(self):
-        return self.__to_str()
 
-    def __repr__(self):
-        return self.__to_str()
+class InstanceSegmentationResult(NamedTuple):
+    segmentedObjects: List[Union[SegmentedObject, SegmentedObjectWithRects]]
+    # Contan per class saliency_maps and "feature_vector" model output if feature_vector exists
+    saliency_map: List[np.ndarray]
```

Comment:
In other Result objects saliency_map is just an np.ndarray. In OTX it's a list, but here we can unify the representation. Is there any particular reason to use list here?

Reply:
Missing classes are represented not as  `np.ndarray` in that list. Should I fill the corresponding axes with zeros?